services/funfuzz/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mozillasecurity/fuzzos:latest
+
+LABEL maintainer Gary Kwong <gkwong@mozilla.com>
+WORKDIR /home/worker/funfuzz
+
+USER root
+COPY recipes /tmp/recipes
```

Comment:
Add empty line here. It gets overlooked that we copy all `recipes` to `/tmp/recipes`.